src/NHPersistenceCompatibilityTests/SharedDefinitions/RawPersister.cs

```diff
@@ -8,25 +9,54 @@ public void Save(string typeFullName, string body, string correlationPropertyNam
         var sagaDataType = Type.GetType(typeFullName);
         var sagaData = JsonConvert.DeserializeObject(body, sagaDataType);
 
-        var persister = Activator.CreateInstance(Type.GetType("Persister"));
-
-        var saveMethod = persister.GetType().GetMethod(nameof(Save)).MakeGenericMethod(sagaDataType);
+        var persister = CreatePersisterInstance();
+        var saveMethod = GetMethod(persister, nameof(Save), sagaDataType);
 
         saveMethod.Invoke(persister, new [] { sagaData, correlationPropertyName, correlationPropertyValue });
     }
 
-    public object Get(string typeFullName, Guid id)
+    public void Update(string typeFullName, string body)
     {
         var sagaDataType = Type.GetType(typeFullName);
+        var sagaData = JsonConvert.DeserializeObject(body, sagaDataType);
 
-        var persister = Activator.CreateInstance(Type.GetType("Persister"));
+        var persister = CreatePersisterInstance();
+        var updateMethod = GetMethod(persister, nameof(Update), sagaDataType);
 
-        var getMethod = persister.GetType().GetMethod(nameof(Get)).MakeGenericMethod(sagaDataType);
+        updateMethod.Invoke(persister, new[] { sagaData });
+    }
+
+    public object Get(string typeFullName, Guid id)
+    {
+        var sagaDataType = Type.GetType(typeFullName);
+
+        var persister = CreatePersisterInstance();
+        var getMethod = GetMethod(persister, nameof(Get), sagaDataType);
```

Comment:
I was looking at that more like on convention. `Persister`s from different versions should implement methods which are named the same as those of `RawPersister`. 

I've switched to strings to make it make sure it's more visible that names come from `Persister` types.